cmd/orchestrator.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/thirdweb-dev/indexer/internal/common"
+	"github.com/thirdweb-dev/indexer/internal/orchestrator"
+)
+
+var (
+	orchestratorCmd = &cobra.Command{
+		Use:   "orchestrator",
+		Short: "TBD",
+		Long:  "TBD",
```

Comment:
```suggestion
		Short: "o",
		Long:  "orchestrator",
```

Reply:
These are descriptions for the command, not shorthand syntax